x-pack/plugins/actions/README.md

```diff
@@ -106,27 +106,27 @@ Built-In-Actions are configured using the _xpack.actions_ namespoace under _kiba
+Uniquely, the _PagerDuty Action Type_ has been configured to support the service's Events API (at _https://events.pagerduty.com/v2/enqueue_, which you can read about [here](https://v2.developer.pagerduty.com/docs/events-api-v2)) as a default, but this too, must be included in the hostsAllowList before the PagerDuty action can be used.
 
 ### Configuration Utilities
 
 This module provides a Utilities for interacting with the configuration.
 
 | Method                    | Arguments                                                    | Description                                                                                                                                                                                                                                                                                                 | Return Type                                           |
 | ------------------------- | ------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
-| isWhitelistedUri          | _uri_: The URI you wish to validate is whitelisted           | Validates whether the URI is whitelisted. This checks the configuration and validates that the hostname of the URI is in the list of whitelisted Hosts and returns `true` if it is whitelisted. If the configuration says that all URI's are whitelisted (using an "\*") then it will always return `true`. | Boolean                                               |
-| isWhitelistedHostname     | _hostname_: The Hostname you wish to validate is whitelisted | Validates whether the Hostname is whitelisted. This checks the configuration and validates that the hostname is in the list of whitelisted Hosts and returns `true` if it is whitelisted. If the configuration says that all Hostnames are whitelisted (using an "\*") then it will always return `true`.   | Boolean                                               |
+| isAllowListedUri          | _uri_: The URI you wish to validate is allowed               | Validates whether the URI is allowed. This checks the configuration and validates that the hostname of the URI is in the list of allowed Hosts and returns `true` if it is allowed. If the configuration says that all URI's are alloed (using an "\*") then it will always return `true`. | Boolean                                               |
```

Comment:
Like the config name, something about these method names doesn't seem quite right.

One suggestion: `isUriAllowed()`, `isHostnameAllowed()`, `ensureUriAllowed()`, `ensureHostnameAllowed()`